docs/containers-authfile.5.md

```diff
@@ -0,0 +1,43 @@
+% containers-authfile(5)
```

Comment:
To be consistent with the naming scheme of other man pages, I prefer calling it `containers-auth.json` (see `containers-storage.conf`, `containers-registries.conf` etc.)

Reply:
SGTM

Reply:
fixed in the next push.  I've also changed the file name so I think these comments may disappear.